volume/core.go

```diff
@@ -29,16 +31,25 @@ type Interface interface {
 	VolumeOpen(ctx context.Context, path string) (plugin.SizedReader, error)
 	// Accepts a path and streams updates to the content associated with that path.
 	VolumeStream(ctx context.Context, path string) (io.ReadCloser, error)
+	// Deletes the volume node at the specified path. Mirrors plugin.Deletable#Delete
+	VolumeDelete(ctx context.Context, path string) (bool, error)
```

Comment:
It's unlikely that VolumeDelete will return false, but the general case is easy enough that we might as well support that capability if people ever want to use it. Also maintains symmetry w/ plugin.Deletable#Delete.